unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls

```diff
@@ -0,0 +1,199 @@
+        if(populateConversionRTs){
+            orgSettings.hed__Lead_Converted_Account_RTypes__c = hed.UTIL_Describe_API.getAdminAccRecTypeID() + ';' +
+                hed.UTIL_Describe_API.getHhAccRecTypeID(); //populate Admin & Household account conversion types
+        } else{
+            orgSettings.hed__Lead_Converted_Account_RTypes__c = null; //ensure no Account record types for LC            
+        }
+        orgSettings.hed__Admin_Account_Naming_Format__c = '{!{!FirstName}} {!LastName} Administrative Account';
+        orgSettings.hed__Household_Account_Naming_Format__c = '{!FirstName} {!LastName} Household';
+        upsert orgSettings;
+        hed.UTIL_CustomSettings_API.getSettingsForTests(orgSettings);
+        return orgSettings;
+    }
+
+
+    /**
+     * @description T-5026176: Verify that the Lead Conversion Account Naming has no changes with the RT setting disabled
+     * (ie, Account Name should be the same as the Lead Company)
+    */
+    @isTest
+    private static void convertLeadVerifySameAccountNameWithoutSettings() {
+        hed__Hierarchy_Settings__c orgSettings = createSettings(false);
+
+        Lead testLead = hed.UTIL_UnitTestData_API.getLead();
+        testLead.Company = testLead.company + Datetime.now().getTime();
+        insert testLead;
+
+        Test.startTest();
+        Database.LeadConvert lc = new Database.LeadConvert();
+        lc.setLeadId(testLead.Id);
+
+        LeadStatus convertedStatus = [SELECT Id, MasterLabel FROM LeadStatus WHERE IsConverted = TRUE LIMIT 1];
+        lc.setConvertedStatus(convertedStatus.MasterLabel);
+
+        Database.LeadConvertResult lcr = Database.convertLead(lc);
+        Test.stopTest();
+        System.assert(lcr.isSuccess()); //Assert conversion succeeded
+
+        //Verify Account Name is the same as the Lead Company
+        List<Account> accts = [SELECT Id, Name FROM Account WHERE Name = :testLead.Company];
+        System.assertEquals(1, accts.size(), 'There should be one Account with the same name as the Lead.Company.');
+    }
+
+    
+    /**
+     * @description T-5026177: Verify that the Lead Conversion Account Naming has no changes with the Record Type 
+     * setting enabled. (Lead Conversion will create a new Account with the Profile's default Account, so ensure 
+     * the default is different than the hed__Lead_Converted_Account_RTypes__c setting.)
+    */
+    @isTest
+    private static void convertLeadVerifySameAccountNameWithSettings() {
+        //Set Academic Account Record Type as the RT setting since it is not the default Account RT
+        hed__Hierarchy_Settings__c orgSettings = createSettings(false);
+        orgSettings.hed__Lead_Converted_Account_RTypes__c = hed.UTIL_Describe_API.getAcademicAccRecTypeID();
+
+        //Confirm Academic is not the default RT (default will be used for the new Account during Lead Conversion)
+        Schema.DescribeSObjectResult dsr= Schema.SObjectType.Account; 
```

Comment:
Spacings look weird for these lines. Not sure why prettier isn't complaining.

Reply:
Also, quick question on this. Since record types are visible to apex tests, will this test method fail if default record type is set to Academic? Will it break integration or customers' org? I guess probably fine for unpackaged tests thou.

Reply:
Good consideration, but I think it will be fine for unpackaged tests.